shardingsphere-infra/shardingsphere-infra-common/src/main/java/org/apache/shardingsphere/infra/config/properties/ConfigurationPropertyKey.java

```diff
@@ -132,7 +132,12 @@
      * Proxy backend query fetch size. A larger value may increase the memory usage of ShardingSphere Proxy.
      * The default value is -1, which means set the minimum value for different JDBC drivers.
      */
-    PROXY_BACKEND_QUERY_FETCH_SIZE("proxy-backend-query-fetch-size", "-1", int.class);
+    PROXY_BACKEND_QUERY_FETCH_SIZE("proxy-backend-query-fetch-size", "-1", int.class),
+    
+    /**
+     * Whether check single table.
+     */
+    CHECK_SINGLE_TABLE_ENABLED("check-single-table-enabled", String.valueOf(Boolean.FALSE), boolean.class);
```

Comment:
how about 'allow tables with the same name exist '? Or something like that.

Reply:
> how about 'allow tables with the same name exist '? Or something like that.

@tristaZero `allow-duplicate-single-table`?

Reply:
@tristaZero Changing to `allow...` will cause the default value to be `true`, and when user checks the single table, it needs to be set to `false`, which is the opposite of using other parameters. Is it better to change to `check-single-table-duplicate-enabled`?